plugins/bcli.c

```diff
@@ -431,6 +432,26 @@ static struct command_result *process_getblockchaininfo(struct bitcoin_cli *bcli
 	return command_finished(bcli->cmd, response);
 }
 
+static struct command_result *process_getrawtransaction(struct bitcoin_cli *bcli)
+{
+	const jsmntok_t *tokens = NULL;
+	struct json_stream *response;
+
+	tokens = json_parse_simple(bcli->output,
+				   bcli->output, bcli->output_bytes);
+	if (!tokens) {
```

Comment:
We usually elide `{` `}` for single-statement `if`/`while`/`for`.

Reply:
I mimicked the style from `*process_getblockchaininfo` and `*process_getutxout` etc.